evm-contracts/src/v0.6/dev/PrepaidAggregator.sol

```diff
@@ -680,4 +708,9 @@ contract PrepaidAggregator is AggregatorInterface, Owned, WithdrawalInterface {
     _;
   }
 
+  modifier onlyWhitelistedRequesters() {
```

Comment:
Then also to match the existing Aggregator: `ensureAuthorizedRequester`.

Reply:
We started a naming pattern here which has carried over `only`, and I'd like to stick with that moving forward. Also it's internal.